components/putty/patches/02-cmdgen.patch

```diff
@@ -0,0 +1,69 @@
+--- putty-0.70/cmdgen.c	2017-07-04 21:30:55.000000000 +0000
++++ cmdgen.c	2017-08-30 04:57:16.469173728 +0000
+@@ -236,8 +236,8 @@
```

Comment:
What are we changing here and why?

Reply:
Apparently there is a C Variable called Private which is used in other places somewhere. GCC complained about double Declared Variable. Thus I renamed it to PPRIVATE meaning PuttyPrivate so that putty compiles.

Reply:
sorry, I haven't spotted double P